src/entities/hubdb.js

```diff
@@ -240,6 +272,34 @@ export default function hubdbApi(baseOptions) {
      * @returns {Promise}
      */
     updateTableRow,
+    /**
+     * Batch update in a HubDB table
+     * @async
+     * @memberof hs/hubdb
+     * @method updateTableRow
+     * @param {int} tableId
+     * @param {object} options
+     * @param {array} rows The rows to update
+     * @example
+     * const hs = new HubSpotClient(props);
+     * const options = {
+     * rows: [
+     * {
+        id: 1234567,
+        createdAt: 1000000000,
+        path: 'test',
+        name: 'Test',
+        values: {
+          '2': 'Some data',
+          '3': 'None',
+        }
+      },
+      ...more rows
+      ]}
+     * hs.hubdb.updateTableRow(tableId, options).then(response => console.log(response))
```

Comment:
Should this be called `updateTableRows` given that it accepts an array?

Reply:
My bad, it's a typo, I'll fix it on the next commit

Reply:
Typo is now fixed!